### PR TITLE
First cut of Sonar Cloud integration

### PR DESCRIPTION
**Pull request by alvarosanchez**

### CODE REVIEW

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -49,12 +49,13 @@ jobs:
           [ -f ./setup.sh ] && ./setup.sh || true
       - name: Build with Gradle
         run: |
-          ./gradlew dependencyUpdates check --no-daemon --parallel --continue
+          ./gradlew dependencyUpdates check sonarqube --no-daemon --parallel --continue
```
Comment (graemerocher) [inline .github/workflows/gradle.yml]: Can we run this in a separate workflow?

Comment (alvarosanchez) [reply .github/workflows/gradle.yml]: Sure!

**Review by graemerocher — APPROVED** [review]

**Review by graemerocher — APPROVED** [review]

**Comment by jameskleeh** [comment]
No idea why its showing all of these commits. Did you base this from 3.4?

**Comment by alvarosanchez** [comment]
No, 3.3.x. Is 3.3.x what it should target?

**Comment by jameskleeh** [comment]
It can target 3.3, but it shouldn't show all of these extra commits. In the past when this happened it led to commits being reverted silently. Can you try rebasing onto 3.3.x?

**Comment by alvarosanchez** [comment]
Done.

**Pull request merged**